src/net/sourceforge/kolmafia/session/ChoiceAdventures.java

```diff
@@ -2797,12 +2803,32 @@ public static final ChoiceOption findOption(final ChoiceOption[] options, final
         new ChoiceOption("fight"),
         SKIP_ADVENTURE);
 
-    // Choice 513 is Staring Down the Barrel
-    // -> can skip if have +20 cold damage
-    // Choice 514 is 1984 Had Nothing on This Cellar
-    // -> can skip if have +20 stench damage
-    // Choice 515 is A Rat's Home...
-    // -> can skip if have +20 spooky damage
+    // Staring Down the Barrel
+    new ChoiceAdventure(
+        513,
+        "Woods",
+        "Typical Tavern",
+        // Option...
+        new ChoiceOption("3-5 ice-cold Willers"),
+        new ChoiceOption("Get rid of crate without spending and adventure"));
+
+    // 1984 Had Nothing on This Cellar
+    new ChoiceAdventure(
+        514,
+        "Woods",
+        "Typical Tavern",
+        // Option...
+        new ChoiceOption("3-5 rat whiskers and smiling rat (once)"),
```

Comment:
```suggestion
        new ChoiceOption("3-5 rat whiskers and smiling rat (sometimes)"),
```

specifically, you have a chance to get a smiling rat if you don't already have one (in inventory or terrarium)

Reply:
done.